nestjs/src/notifications/notifications.service.ts

```diff
@@ -91,7 +93,7 @@ export class NotificationsService {
     });
   }
 
-  public buildChannelMessage(channel: NotificationChannelSettings & { externalId: string }, data: object) {
+  public buildChannelMessage(channel: NotificationChannelSettings & { externalId?: string }, data: object) {
```

Comment:
curious, what is the use case? seems this should be catched on upper calls

Reply:
yes, not needed after rebase. removed